projects/solar/src/test_data_tx.c

```diff
@@ -0,0 +1,93 @@
+#include "can.h"
+#include "can_msg_defs.h"
+#include "can_transmit.h"
+#include "data_store.h"
+#include "data_tx.h"
+#include "event_queue.h"
+#include "gpio.h"
+#include "interrupt.h"
+#include "log.h"
+#include "ms_test_helpers.h"
+#include "soft_timer.h"
+#include "solar_events.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+int count = -1;                 // count of can msgs sent, incremented in rx_handler
+bool changeable_is_set = true;  // value that tells data_tx if data points are set
```

Comment:
These should be static and prefixed with `s_`. I'm also not too big a fan of the comments, I'd prefer more descriptive names like e.g. `can_msg_count` instead of the comment describing what the variable does (unless it's more complex).

Reply:
Also, `count` should be a fixed-width type, probably `uint16_t`.